src/history/api.py

```diff
@@ -3,16 +3,17 @@
 from datetime import datetime
 from pathlib import Path
 from typing import Union
+import re
```

Comment:
src/history/api.py:1:1: I001 [*] Import block is un-sorted or un-formatted

Would be nice if you could alphasort this (place this under `functools`, and above `sqlite3`.)